mmdeploy/codebase/mmpose/deploy/pose_detection_model.py

```diff
@@ -4,11 +4,22 @@
 import mmcv
 import numpy as np
 import torch
+from mmcv.utils import Registry
 
 from mmdeploy.codebase.base import BaseBackendModel
-from mmdeploy.utils import Backend, get_backend, load_config
+from mmdeploy.utils import (Backend, get_backend, get_codebase_config,
+                            load_config)
 
 
+def __build_backend_model(cls_name: str, registry: Registry, *args, **kwargs):
+    return registry.module_dict[cls_name](*args, **kwargs)
+
+
+__BACKEND_MODEL = mmcv.utils.Registry(
+    'backend_text_recognizer', build_func=__build_backend_model)
```

Comment:
'backend_text_recognizer' is inappropriate.

Reply:
changed to backend_pose_detectors